os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c

```diff
@@ -953,7 +957,12 @@ u32 app_mpu_nocache_init(void)
 	if (mpu_cfg.region_size >= 32) {
 		mpu_region_cfg(mpu_entry, &mpu_cfg);
 	}
-
+#ifndef CONFIG_PLATFORM_TIZENRT_OS
+/*	For TizenRT  below 3 MPU seting is not use
+	"close 216K irom_ns cache" and "close 80K drom_ns cache" is to prevent ROM code
+	to access data through Cache, by testing to remove it perforamnce have not drop.
```

Comment:
performance*

Reply:
Also, do you mean that :
by removing these 3 settings, performance does not drop ?

Reply:
Yes, there is not much performance drop after removing these 3 MPU setting.
I will rephrase it later.